pkg/validation/limits.go

```diff
@@ -118,6 +118,9 @@ type Limits struct {
 	// RecordingRules allow to specify static recording rules. This is not compatible with recording rules
 	// coming from a RecordingRulesClient, that will replace any static rules defined.
 	RecordingRules RecordingRules `yaml:"recording_rules" json:"recording_rules" category:"experimental" doc:"hidden"`
+
+	// SymbolizerEnabled enables the symbolizer in the query frontend.
+	SymbolizerEnabled bool `yaml:"symbolizer_enabled" json:"symbolizer_enabled" doc:"hidden"`
```

Comment:
is it not possible to enable this flag wit cli? is this on purpose?

Reply:
Should we enable per tenant flag from cli? In symbolizer.go we have defined:
```go

func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
    f.BoolVar(&cfg.Enabled, "symbolizer.enabled", false, "Enable symbolization for unsymbolized profiles")
    ...
}
```